### ISSUE TITLE
nuked nodes could cause intervals to be misaligned for peers attempting to pullsync with the nuked peer

### ISSUE THREAD

**Issue by istae** [ethersphere/bee#3821]
⚠️ Support requests in an issue-format will be closed immediately. For support, go to Swarm's [Discord](https://discord.gg/XGNBEKktkD).

### Summary
When a node is nuked, it's binIds are also nuked, and after the node reboots from a nuke, the binIDs start from zero.
Peers that restart syncing with the nuked nodes will miss the certain intervals of chunks because the locally stored sync intervals will be ahead of the nuked binIDs

### Expected behavior
Syncing peers should reset their own intervals when they detect that the peers has been nuked.

### Possible solution
The nodes can store a timestamp of when they first starting running. 
When ever there is a change in that timestamp when peers exchange cursors, the peer can reset the sync intervals of that peer.

**Comment by istae** [comment]
the nuke, with the new the localstore, should not remove the binIDs, but remove everything else.

**Comment by istae** [comment]
ALSO: consider removing the nuke command all together!

**Comment by ldeffenb** [comment]
Removing the db nuke would probably just result in folks preserving the keys and deleting all other files.  At least with nuke, the node is somewhat in control of what gets deleted and what gets preserved.

**Comment by istae** [comment]
Related #3844

**Comment by ct-ctrl** [comment]
5/6/23 - backlog refinement conversation summation: This is no longer needed, but the decided improvement is a 'self repairing mechanism' whereby pullsync resyncs the intervals or complete historical resync periodically (every day). This should also be randomised that this event is not happening in parallel en masse.
